### PR TITLE
[FIX] website_forum: fix image display in a forum post

### PR DESCRIPTION
**Pull request by younn-o**
The container size of a forum post was not correct when an image was
displayed with right or left alignment. Also, large images were
overflowing the post.

The css property clear was added at the end of the posts so that
floating elements do not float over the next section.

task-2469516

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_forum/static/src/scss/website_forum.scss [hunk 1]
```diff
@@ -66,6 +66,16 @@ $bronze: #eea91e;
         p {
             margin-bottom: 0.5rem;
         }
+
+        img {
+            max-width: 100%;
+        }
```
Comment (bvr-odoo) [inline addons/website_forum/static/src/scss/website_forum.scss:72]: we have already a rule to do this
```
    .forum_answer img, .question-block img {
        max-width: 100%
    }
```
but the class `question-block` is no longer used, so the origin of the bug is here
with your fix, you duplicate the rule on forum_answer img...
So I guess you could simply remove the code I pasted here and only keep the rule you added on `o_wforum_readable img`

**Review by bvr-odoo — CHANGES_REQUESTED** [review]
I see that the bug is already present in V12
So you can also retarget your branch on this version

File: addons/website_forum/static/src/scss/website_forum.scss (hunk 1 above)
Comment (younn-o) [reply addons/website_forum/static/src/scss/website_forum.scss:72]: Another fix for 12.0 has been pushed on https://github.com/odoo/odoo/pull/70489

**Review by bvr-odoo — APPROVED** [review]

**Review by qsm-odoo — APPROVED** [review]
@robodoo rebase-ff r+

**Pull request closed without merging**